assets/js/googlesitekit/data/utils.js

```diff
@@ -20,6 +20,8 @@
  * External dependencies
  */
 import invariant from 'invariant';
+import mapValues from 'lodash/mapValues';
+import memize from 'memize';
```

Comment:
Note that while `memize` was added to `package.json`, it's already a dependency of `@wordpress/data` so this just makes it safe to use without really adding anything to the bundle.